### ISSUE TITLE
Give a Donation Address in About that Matches the Coin Being Used

### ISSUE THREAD

**Issue by jjeffryes** [OpenBazaar/openbazaar-desktop#1188]
The About/Donations section currently has a hard-coded bitcoin address and QR code.

When BCH and ZCash addresses are ready, show an address that matches the coin the user's server is using.

**Comment by rmisio** [comment]
I think we decided a few weeks back that we would actually show all donation addresses. This was waiting on @morebrownies to design how that would look. He was busy at the time and this likely got lost.

**Comment by morebrownies** [comment]
I think it's probably best to show all addresses all the time. People could still donate to them from their mobile device (or 3rd party wallets) if they wanted to. 

We'll only want to show the "open in wallet" button next to the active currency the user has selected.

**Comment by jjeffryes** [comment]
@morebrownies I think we may need a design on this one, showing two QR codes on the page could be awkward looking.

**Comment by morebrownies** [comment]
Please add ticket in Jira. My queue is pretty deep at this point.

**Comment by jjeffryes** [comment]
@morebrownies I'll put one in the backlog, this is not a priority, I just wanted to flag it as needing design so we remembered.

**Comment by jjeffryes** [comment]
This is effectively done, about now shows the two available donation addresses (in the multi-wallet branch).
